app/views/course/leaderboards/show.html.slim

```diff
@@ -1 +1,43 @@
-p To be implemented later
+= page_header t('course.leaderboard.index.header')
```

Comment:
What I suggest you can do is stick with convention first, next PR you can change it to be a singular controller.